types/tomcat_connectors.pp

```diff
@@ -0,0 +1 @@
+type Jira::Tomcat_connectors = Hash[Stdlib::Port::Unprivileged, Jira::Tomcat_attributes]
```

Comment:
Could you add unit tests for the types? Here's an example https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/spec/type_aliases/yes_no_spec.rb

You just have two arrays with things that match or don't.

Reply:
Ok, will add tests for the types today or tomorrow, hopefully.

Reply:
Wow, that "today or tomorrow" was more than a week ago. Life and work got in the way but I haven't forgotten about this.

Reply:
I have now added spec tests for the type aliases and Travis is currently running the tests.